### PR TITLE
[BE]: Database Model

### PR DESCRIPTION
**Pull request by AzizRahmawan**

### CODE REVIEW

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,126 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
```
Comment (affantaufiqur) [inline server/prisma/schema.prisma:9]: pake postgre?

**Review by affantaufiqur — COMMENTED** [review]
id di table nya pake ulidx/uuid aja, jangan autoincrement

File: server/prisma/schema.prisma (hunk 1 above)
Comment (AzizRahmawan) [reply server/prisma/schema.prisma:9]: oh iya salah

Comment (AzizRahmawan) [reply server/prisma/schema.prisma:9]: berarti gausah auto default?

Comment (AzizRahmawan) [reply server/prisma/schema.prisma:9]: gini id String @default(uuid())?

Comment (affantaufiqur) [reply server/prisma/schema.prisma:9]: coba kaya gini
```prisma
model User {
  id   String @id @default(uuid())
  name String
}
```
https://www.prisma.io/docs/orm/reference/prisma-schema-reference#uuid

Comment (AzizRahmawan) [reply server/prisma/schema.prisma:9]: okok

**Review by affantaufiqur — APPROVED** [review]
@AzizRahmawan ok, merge sendiri yo

**Pull request merged**